client/trino-client/src/main/java/io/trino/client/StatementClient.java

```diff
@@ -41,8 +42,11 @@ public interface StatementClient
 
     QueryStatusInfo currentStatusInfo();
 
+    // For backward compatibility and migration path
     QueryData currentData();
 
+    @Nonnull ResultRows currentRows();
```

Comment:
drop `@Nonnull` this is default

Reply:
IntelliJ underlines this as non being annotated

Reply:
So change your intellij config.
Nonnull is default in Trino codebase and we do not use it.